src/pybind/mgr/dashboard/constraints.txt

```diff
@@ -5,3 +5,5 @@ python3-saml~=1.4
 requests~=2.26
 Routes~=2.4
 cheroot~=10.0
+grpcio~=1.46
+grpcio-tools~=1.46
```

Comment:
Did you mean to make dashboard requirements changes in these commits? The message `[DNM] Added PR to fix fc issue.` is rather cryptic to me.

Reply:
This is related to: https://github.com/ceph/ceph/pull/55913
I had more info in the commit, but each push would retrigger "mention" in that PR. I'll make sure to have this removed before it leaves draft.

Reply:
BTW if you ever would like to try building for a different distro w/o setting up a full blown VM, I've been trying to get more people looking at and testing out my PR #59841